pkg/webtest/web_server_test.go

```diff
@@ -80,5 +80,43 @@ func TestWebServer(t *testing.T) {
 }
 
 func TestProxy(t *testing.T) {
+	mux := http.NewServeMux()
```

Comment:
This test case is duplicate to `TestWebServer`, so I prefer to remove `TestProxy` instead of having many duplicate code. WDYT?

Reply:
Sure, I think we can remove `TestProxy`. No problem, I'll make this change.